examples/1-getting-started/src/app/components/Todo.tsx

```diff
@@ -23,9 +23,9 @@ export const Todo: FC<Props> = props => {
 Todo.displayName = 'Todo';
 
 const RemoveTodo = `
-mutation($id: ID!) {
-  toggleTodo(id: $id) {
-    id
+  mutation($id: ID!) {
```

Comment:
Is this prettier?

Reply:
That will probably have been me, I dislike the indentation like that but prettier didn't complain/correct it.

Reply:
If prettier doesn't complain, neither do I!